Managed data views permissions and duplication instructions (#3370)
elastic/docs-content: explore-analyze/find-and-organize/data-views.md

  - `Data View Management` {{kib}} privilege.
  - `view_index_metadata` {{es}} privilege.
* If a read-only indicator appears, you have insufficient privileges to create or save {{data-sources}}. In addition, the buttons to create {{data-sources}} or save existing {{data-sources}} are not visible.

> * {applies_to}`stack: ga 9.2` Some data views are exclusively configured and **managed** by Elastic. You can view and use these managed data views, but you can't edit them. If you'd like to use a modified version of a managed data view, you can [duplicate it](#duplicate-managed-data-view) and edit that new copy as needed.

Reviewer: Maybe it's redundant to say "configured and managed"? I think configuration is part of management, or close enough. What about this? 
```suggestion
* {applies_to}`stack: ga 9.2` Some default data views are managed by Elastic. You can't edit or delete them, but you can [copy](#duplicate-managed-data-view) them and edit the copies.
```
I may have gotten a bit carried away on the wordsmithing here, feel free to ignore 😄
